packages/next/src/generators/react-query/utils/eslint.ts

```diff
@@ -0,0 +1,22 @@
+import { mergeEslintConfigs, updateEslintConfig } from '@ensono-stacks/core';
+import { Tree } from '@nx/devkit';
+import { Linter } from 'eslint';
+
+import { installDependencies } from './dependancies';
+import { ReactQueryGeneratorSchema } from '../schema';
+
+const reactQueryESLintConfig: Linter.Config = {
+    extends: ['plugin:@tanstack/eslint-plugin-query/recommended'],
+};
+
+export const updateESLint = (
+    tree: Tree,
+    projectRootPath: string,
+    options: ReactQueryGeneratorSchema,
+) => {
+    updateEslintConfig(tree, projectRootPath, baseConfig => {
+        return mergeEslintConfigs(baseConfig, reactQueryESLintConfig);
+    });
+
+    return installDependencies(tree, options);
```

Comment:
I had to search for this line of code to check whether/where you were installing the required dependencies. 
Didn't feel entirely obvious that updateEsLint would bear the responsibility for installing dependencies. 
Recommend adding installDependencies to the task list in 'packages/next/src/generators/react-query/generator.ts'

Reply:
In made sense in my head that due to the only installation being eslint that this can live in that file. Do we have a standardised way of making generators? If so I an happy to adhere to that

Reply:
I wouldn't say we have a standardised way as such, just wasn't entirely obvious to myself where it was actually being called as I could see that you had an import for installDependencies in the generator (unused). 

In other generators I think we just have a function like so which is passed into the runTasksInSerial. Personal preference is to keep the install within the generator, as if we add any new ones in the future, unrelated to eslint, then future developers may not notice the hidden install in eslint.ts, resulting in the install occuring twice
```
function updateDependencies(tree) {
    return addDependenciesToPackageJson(
        tree,
        {},
        {
            plugin: X_VERSION,
        },
    );
} 
```